packages/kysely/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Kysely, ParseJSONResultsPlugin, PostgresDialect, DeduplicateJoinsPlugin } from "kysely";
+import { Pool } from "pg";
+
+import type { DB, Booking } from "./types";
+
+export type { DB, Booking };
+
+const connectionString = process.env.DATABASE_URL ?? "postgresql://postgres:@localhost:5450/calendso";
+
+const pool = new Pool({ connectionString });
+
+// 3. Create the Dialect, passing the configured pool instance
+const dialect = new PostgresDialect({
+  pool: pool, // Use the pool instance created above
+});
+
+// 4. Create the Kysely instance as before
+const db = new Kysely<DB>({
```

Comment:
Do we need to make this a singleton like we do with prisma right now? cc @zomars @emrysal

Reply:
I don't know the actual implications of doing it this way with Kysely. Maybe we can use globalThis like we do with prisma.

```suggestion
const db = globalThis.db ?? new Kysely<DB>({
  dialect,
  plugins: [new ParseJSONResultsPlugin(), new DeduplicateJoinsPlugin()],
});

if (!globalThis.db) {
  globalThis.db = db;
}
```